src/widget/city_overlay_health.c

```diff
@@ -26,7 +26,7 @@ static int show_building_hospital(const building *b)
 
 static int show_building_sickness(const building *b)
 {
-    return b->type == BUILDING_HOSPITAL || b->type == BUILDING_DOCTOR;
+    return b->type == BUILDING_HOSPITAL || b->type == BUILDING_DOCTOR || b->type == BUILDING_BARBER || b->type == BUILDING_BATHHOUSE;
```

Comment:
These buildings and their walkers don't seem to affect sickness, so they shouldn't appear on the sickness overlay.

Reply:
I don't know if this works actually, because on latest unstable, sickness overlay only shows clinics and hospitals.

But Barbers and Bathhouses _should_ be shown too, as they affect city global health rating,  and this rating affects sickness decrease. So, the sickness overlay would shos everything related to sickness, to have a "global" view:
- buildings with sickness (trading docks, warehouses, granaries, houses)
- sickness spreaders (caravan merchants, dockers, cartpushers from infected buildings)
- all health facilities (clinics, hospitals, barbers, and bathhouses). Though, if possible, not show their walkers (useless information on sickness overlay), only their buildings.

Reply:
Right now, the only building that affects global city health is the clinic. City health value is pretty much the percentage of how many houses are covered by clinics. Barbers, hospitals and bathhouses have no effect. This probably should change, and when it does showing them on the sickness overlay will make sense.

Reply:
ok I updated the ratio calculation

Reply:
> Right now, the only building that affects global city health is the clinic. 

Geez, I didn't know that, I thought it was all buildings shown in the Health Advisor panel. ^^